pds/contracts/PackNFT_TopShot.cdc

```diff
@@ -439,6 +439,31 @@ access(all) contract PackNFT: NonFungibleToken, IPackNFT {
                     })
                 )
                 return collectionData
+            case Type<MetadataViews.NFTCollectionDisplay>():
+                let bannerImage = MetadataViews.Media(
+                    file: MetadataViews.HTTPFile(
+                        url: "https://nbatopshot.com/static/img/top-shot-logo-horizontal-white.svg"
+                    ),
+                    mediaType: "image/svg+xml"
+                )
+                let squareImage = MetadataViews.Media(
+                    file: MetadataViews.HTTPFile(
+                        url: "https://nbatopshot.com/static/img/og/og.png"
```

Comment:
i don't think this image is correct https://github.com/dapperlabs/nba-smart-contracts/pull/267/files, context: https://dapperlabs.slack.com/archives/CTGAW0TM2/p1726627172325759

Reply:
thanks! updated